.github/workflows/stacks-blockchain-api.yml

```diff
@@ -452,28 +459,55 @@ jobs:
             semantic-release-slack-bot
             @semantic-release/exec
 
-      - name: Build/Tag/Push Image
-        uses: docker/build-push-action@v1
+      - name: Docker Meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            blockstack/${{ github.workflow }}
+            hirosystems/${{ github.workflow }}
+          tags: |
+            type=ref,event=branch
+            type=semver,pattern=${{ steps.semantic.outputs.new_release_version }}
+            type=sha
```

Comment:
Not sure this will work for either image, I think the `pattern` is invalid, and it won't create a docker tag for PRs. [We typically configure it like this](https://github.com/hirosystems/gh-actions-example/blob/main/.github/workflows/ci.yml#L80-L83) to account for pr's, branches, semantic versions and a shorter version. You can tack on the `type=sha` too if you want to keep that.

Reply:
> Thanks, I'll try that. If I make this change I still need to keep the "Semantic Release" step because it affects other things, right?

Yes, the Semantic Release step is the step actually performing the release. This meta step is only forming a string in the correct semver syntax than can be used when pushing up a docker tag.